go.mod

```diff
@@ -220,8 +220,8 @@ require (
 	github.com/pkg/term v1.1.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/posener/complete v1.2.3 // indirect
-	github.com/pulumi/pulumi/pkg/v3 v3.166.0
-	github.com/pulumi/pulumi/sdk/v3 v3.166.0
+	github.com/pulumi/pulumi/pkg/v3 v3.167.1-0.20250507103745-1e3ee619cf4d
```

Comment:
Hmm that's not right I wonder what happened here. It broke our automation though. Let me revert quick.